rpi-source

```diff
@@ -332,6 +332,12 @@ info("SoC: %s" % PROCESSOR_TYPES_NAMES[processor_type])
 # FIX usage of -d DEST with relative pathnames
 args.dest = os.path.abspath(args.dest)
 
+if os.path.exists('/boot/firmware/.firmware_revision'):
+    firmware_revision_path = '/boot/firmware/.firmware_revision'
+else:
+    firmware_revision_path = '/boot/.firmware_revsion'
```

Comment:
Typo - `revsion`.

Reply:
Hi,

Fixed in new forced push.

regards,